test/document.test.js

```diff
@@ -11239,3 +11239,12 @@ describe('document', function() {
     assert.equal(doc.mix.val, 'foo');
   });
 });
+
+describe('Check if instance functions that is supplied in schema option is availabe (m0_0a)', function() {
```

Comment:
please remove your iniitials in the title of the unit test describe

```suggestion
describe('Check if instance functions that is supplied in schema option is availabe', function() {
```